build.gradle

```diff
@@ -868,9 +884,9 @@ project('test:system') {
         systemProperty "bookkeeperImageName", System.getProperty("bookkeeperImageName", "bookkeeper") // bookkeeper image name
         systemProperty "zookeeperImageName", System.getProperty("zookeeperImageName", "zookeeper") // zookkeeper image name
         systemProperty "zookeeperImageVersion", System.getProperty("zookeeperImageVersion", "latest") // zookeeper image version
-        systemProperty "imageVersion", System.getProperty("imageVersion") // pravega version.
+        systemProperty "imageVersion", System.getProperty("imageVersion", "latest") // pravega version.
```

Comment:
Should this be `latest`? Shouldn't it be using a specific named version instead?

Reply:
Likely -- I mostly followed suit from the `zookeeperImageVersion` property above. Any suggestions to a sain default?

Reply:
I don't think we should use `latest` as Pravega image here. Ideally `imageVersion` should be pointing to an image that was built from the current source code as this is the intend in most cases - run system tests for the current commit/branch.  If no `imageVersion` provided then system test run should fail

Reply:
Should I just remove the default instead then? Obviously its not possible to determine the exact build tag ahead of time, but if you run the gradle docker task, it produces some sort of version based snapshot `<version>-<hash>-SNAPSHOT` but that build also gets tagged as `latest`, so I think it does cover the main use case.

Reply:
I think it comes to where the build is published after the gradle docker task. If it's published to the same repository as the default one to pull image from for the purpose of  the system test run then it may work. But as I understand it's not always true hence it's better to require the user to explicitly mention what image to use. With that in mind the default image version should be removed and if the user doesn't provide it the system test run will fail.